en/SettingsView/General.md

```diff
@@ -66,42 +67,50 @@ The settings are:
+### Ground Station Location (NMEA GPS Device) {#nmea_gps}
 
-You can change the GPS source using the drop down menu in the NMEA GPS Device field. The configuration options will change based on the selected device (as shown below).
+*QGroundControl* will automatically use an internal GPS to display its own location on the map with a purple `Q` icon (if the GPS provides a heading, this will be also indicated by the icon). It may also use the GPS as a location source for *Follow Me Mode* - currently supported on [PX4 Multicopters only](https://docs.px4.io/en/flight_modes/follow_me.html).
 
-![Nmea GPS Device](../../assets/settings/settings_view_general_gps.jpg)
+If your ground station does not have an internal GPS you can instead connect an external GPS module via a serial or UDP port.
+The device must support the ASCII NMEA format (this is normally the case)
 
-Three sources of GPS positioning for the ground control station are supported:
+Use the *NMEA GPS Device* drop-down selector to manually select the GPS device and other options:
 
-1. **Internal GPS unit:** Many mobile devices have GPS built in. If your device internal GPS is supported, it will be automatically detected and used to display the ground control station position.
-2. **USB GPS unit:** If your device does not have a built in GPS, you may select to use a USB (serial) GPS. The GPS must print ASCII NMEA format (this is normally the case). The serial (COM) port and baudrate for the USB GPS must be configured by the user (see below). If you have trouble with using a USB GPS for positioning, try to disable RTK GPS [auto connection](#auto_connect), close qgc, reconnect your GPS, and open QGC.
-3. **Networked GPS:** If you are using your USB GPS for another application, or you have some other GPS system that is not supported by the other options, you may use a UDP network port to send the GPS data to QGC. The network port for QGC to bind and listen for data must be configured by the user (see below). The data format sent to the UDP port should be in raw ASCII NMEA format.
+- USB connection:
 
-Use these menu items to configure a USB or UDP GPS:
+  ![NMEA GPS Device - Serial](../../assets/settings/general/nmea_gps_serial.jpg)
 
-- **NMEA GPS Device**: Select serial or UDP port for GPS information
-- **NMEA GPS Baudrate**: If serial port is selected, this sets the baudrate for that port
-- **NMEA Stream UDP Port**: If UDP port is selected, this is used to select the port that QGC will listen for the NMEA data (QGC binds the port as a server)
+  - **NMEA GPS Device:** *Serial*
```

Comment:
Is it possible to get an image from a real system for this one? @Williangalvani

Reply:
What do you mean? the hardware? I'm currently using an ftdi attached to a regular gps, haha.
The hardware usually looks like [this one](https://www.amazon.com/Receiver-Antenna-Gmouse-Laptop-Navigation/dp/B073P3Y48Q).
If you mean the UI, there are pictures [in the original PR in QGC].(https://github.com/mavlink/qgroundcontrol/pull/7114)

Reply:
Sorry, I thought enough context from where I added this comment. 

What I mean is that showing how to add a serial connection is not very cool when your example says "none available". There doesn't appear to be an example of this in the original PR.

![image](https://user-images.githubusercontent.com/5368500/53308678-4bb8f180-38f7-11e9-9879-5d8ca78d9a95.png)

Reply:
OK, merging this now. @Williangalvani if you are able to find an appropriate image at some point we can update it. Thanks!